src/pages/community-champions.js

```diff
@@ -273,6 +288,88 @@ const BenefitsAndRewardsItem = ({ icon, title, bullets }) => (
     </BenefitsAndRewardsItemContainer>
 );
 
+const HowToQualifyContainer = styled('div')`
+    margin: ${size.xlarge} 0 ${HOW_TO_QUALIFY_CONTAINER_BOTTOM_MARGIN} 0;
+    @media ${screenSize.upToLarge} {
+        margin: ${size.large} 0;
+    }
+`;
+
+const HowToQualifyContributeTitle = styled(H3)`
+    @media ${screenSize.upToMedium} {
+        margin-bottom: ${size.xsmall};
+    }
+`;
+
+const HowToQualifyContributeSection = styled('div')`
+    margin-bottom: ${size.large};
+    width: ${HOW_TO_QUALIFY_CONTRIBUTE_SECTION_WIDTH};
+    @media ${screenSize.upToMedium} {
+        margin-bottom: ${size.mediumLarge};
+        width: 100%;
+    }
+`;
+
+const HowToQualifyGrid = styled('div')`
+    display: grid;
+    gap: ${size.mediumLarge};
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    @media ${screenSize.upToLarge} {
+        gap: ${size.default};
+        grid-template-columns: repeat(2, minmax(0, 1fr));
+        grid-template-rows: repeat(3, 1fr);
+    }
+`;
+
+const HowToQualifyCardContainer = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.greyDarkThree};
+    border-radius: ${size.xsmall};
+    padding: ${HOW_TO_QUALIFY_CARD_CONTAINER_PADDING};
+    @media ${screenSize.upToLarge} {
+        padding: ${size.mediumLarge};
+    }
+`;
+
+const HowToQualifyCardTitle = styled(H5)`
+    margin: ${size.xsmall} 0;
+    word-wrap: break-word;
+    @media ${screenSize.upToLarge} {
+        margin: ${size.default} 0 ${HOW_TO_QUALIFY_CARD_TITLE_BOTTOM_MARGIN} 0;
+    }
+`;
+
+const HowToQualifyIconContainer = styled('div')`
+    display: flex;
+    height: ${size.xlarge};
+    @media ${screenSize.upToLarge} {
+        height: ${size.large};
+    }
+`;
+
+const HowToQualifyIcon = styled('img')`
+    align-self: center;
```

Comment:
I didn't see this specific css being applied, do you?

Reply:
i do. :o

Reply:
Ohh you are right I was looking at the wrong thing lol, nice!